contracts/modules/staking/StakingPool.sol

```diff
@@ -267,7 +267,7 @@ contract StakingPool is IStakingPool, Multicall {
           ? elapsed * _rewardPerSecond * ONE_NXM / _rewardsSharesSupply
           : 0;
 
-        unchecked { _accNxmPerRewardsShare += newAccNxmPerRewardsShare.toUint96(); }
+        _accNxmPerRewardsShare = uint96(newAccNxmPerRewardsShare + _accNxmPerRewardsShare);
```

Comment:
I wasn't sure about leaving the unchecked blocks. It shouldn't exceed uint96.max since it truncates the value.